lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -1031,7 +1031,7 @@ def build_new_order_xml(action, money, payment_source, parameters = {})
             add_stored_credentials(xml, parameters)
             add_pymt_brand_program_code(xml, payment_source, three_d_secure)
             add_mastercard_fields(xml, payment_source, parameters, three_d_secure) if mastercard?(payment_source)
-            xml.tag! :TokenTxnType, parameters[:token_txn_type] if parameters[:token_txn_type]
+            xml.tag! :TokenTxnType, parameters[:token_txn_type]
```

Comment:
❓ Question:

Given how strict is Orbital XSD definition and that this line will always include `TokenTxnType`, is this working properly with `refund` and `credit` transactions, because those methods are currently using the `build_new_order_xml`.

Reply:
you 're right updated, adding a validation